tss-esapi/tests/integration_tests/error_tests/return_code_tests/resource_manager_tests.rs

```diff
@@ -47,6 +47,12 @@ macro_rules! test_valid_conversion {
             std::stringify!(BaseError::$base_error)
         );
 
+        assert_eq!(
+            BaseError::$base_error,
+            expected_resmgr_rc.base_error(),
+            "base_error() did not return the expected "
```

Comment:
Additional space at the end of the string.

Reply:
It is not a space to much... it is rather a missing word.

Reply:
fixed it. It was suppose to print the expected BaseError.